src/services/download-service.ts

```diff
@@ -69,7 +69,7 @@ class DownloadService {
 
       const downloadUrl = response.url;
 
-      const isContentTypeSupported = this.isContentTypeSupported(response, this.player.isImage() ? 'image' : 'video');
+      const isContentTypeSupported = this.player.isImage() ? true : this.isContentTypeSupported(response);
```

Comment:
assuming you keep using isContentTypeSupported (see comment above)
you can do 
this.player.isImage() || this.isContentTypeSupported(response)